shop3/explicit-stack-search/plan-tree.lisp

```diff
@@ -107,7 +107,7 @@ Particularly useful for structures, but could be generally applicable."
             (or (tree-node-expanded-task d)
                 (tree-node-task d))
             (complex-tree-node-children d))
-    #+ignore(when (tree-node-dependencies d)
```

Comment:
Same as for `primitive-tree-node` `print-object` method... Likely too verbose to print all the dependencies all the time.

IIRC dumping all the children out was also problematic, for the same reason: it means that any time you print a complex node you end up printing its entire subtree with all the dependency links.